logback-classic/src/test/java/ch/qos/logback/classic/spi/ThrowableProxyTest.java

```diff
@@ -158,6 +158,100 @@ public void multiNested() {
         }
         verify(w);
     }
+    
+    @Test
+    public void nullMessage() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/tony19/logback/file/2370881438/issues/source?bid=3222618&fileBranchId=3222653#l163)